content/recipes/findability/updateChemicalIdentities.md

```diff
@@ -0,0 +1,73 @@
+
+From this identifier, other compound identifiers can be determined using the following script:
+
+```
+groovy -identifier <Identifier> -identifierType <IdentifierType>
+```
+This script is available from the chemistry development kit/
+
+
+### 2.2 Updating Wikidata
+With the available identifiers of a compound the compound can be updated within Wikidata using any of the following:
+
+- Direct edits to the compound page from the compound's Wikidata page using the 'edit' functionality 
+- Quick statements at https://quickstatements.toolforge.org/
+
+
+
+## Conclusion
+This recipe discussed several aspects of updating chemical compounds
+- How to retrieve chemical identifiers given an original identifier
+- How to update chemical identification properties in Wikidata 
+
+
+## Authors
+
+
+| Name                                                                                                                                                                                                                                       | Orcid                                                                                                                        | Affiliation                           | Type                                                                              |                                                              Elixir Node                                                              | Credit Role
+|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------|---------------------------------------|-----------------------------------------------------------------------------------|:-------------------------------------------------------------------------------------------------------------------------------------:|:----------------:|
+| <div class="firstCol"><a target="_blank" href='https://github.com/'><img class='avatar-style' src='https://avatars.githubusercontent.com/no_github'></img><div class="d-block">Zachary Warnes</div></a>  </div>         | <a target="_blank" href='https://orcid.org/0000-0000-0000-0000'><i class='fab fa-orcid fa-2x text--orange'></i></a> | Maastricht University     | <i class="fas fa-graduation-cap fa-1x text--orange" alt="Academic"></i> | <img class='elixir-style' src='/the-fair-cookbook/_static/images/logo/Elixir/ELIXIR-UK.svg' ></img> | Writing – Review & Editing, Conceptualization
+
+---
+
+## License
+
+This page is released under the Creative Commons 4.0 BY license.
+
+<a href="https://creativecommons.org/licenses/by/4.0/"><img src="https://mirrors.creativecommons.org/presskit/buttons/80x15/png/by.png" height="20"/></a>
```

Comment:
See this comment: https://github.com/egonw/the-fair-cookbook/pull/9#discussion_r683637430